index.bs

```diff
@@ -559,35 +579,56 @@ Width of the play-area bounds in meters. The bounds are defined as an axis-align
 
-
-## Navigator Interface extension ## {#interface-navigator}
+## VR Interface ## {#interface-vr}
 
 <pre class="idl">
-partial interface Navigator {
-  Promise&lt;sequence&lt;VRDisplay&gt;&gt; getVRDisplays();
-  readonly attribute FrozenArray&lt;VRDisplay&gt; activeVRDisplays;
-  readonly attribute boolean vrEnabled;
+interface VR : EventTarget {
+  Promise&lt;sequence&lt;VRDisplay&gt;&gt; getDisplays();
+  readonly attribute FrozenArray&lt;VRDisplay&gt; activeDisplays;
+  readonly attribute boolean isEnabled;
 };
 </pre>
 
-### Attributes ### {#navigator-attributes}
+### Attributes ### {#vr-attributes}
 
-<dfn method for="Navigator" id="navigator-getvrdisplays-attribute">getVRDisplays()</dfn>
+<dfn method for="VR" id="vr-getdisplays-attribute">getDisplays()</dfn>
 Return a Promise which resolves to a list of available {{VRDisplay}}s. The Promise MUST be rejected if the {{Document}} object is inside an iframe that does not have the {{allowvr}} attribute set.
 
-<dfn attribute for="Navigator" id="navigator-activevrdisplays-attribute">activeVRDisplays</dfn>
-{{activeVRDisplays}} includes every {{VRDisplay}} that is currently presenting.
+<dfn attribute for="VR" id="vr-activedisplays-attribute">activeDisplays</dfn>
+{{activeDisplays}} includes every {{VRDisplay}} that is currently presenting.
+
+<dfn attribute for="VR" id="vr-isenabled-attribute">isEnabled</dfn>
+The {{isEnabled}} attribute's getter must return true if the context object is allowed to use the feature indicated by attribute name {{allowvr}} and VR is supported, and false otherwise.
+
+### Events ### {#vr-events}
+
+The UA MUST provide the following new events. The corresponding events must be of type {{VRDisplayEvent}} and must fire on the {{vr}} object. Registration for and firing of the events must follow the usual behavior of DOM4 Events.
 
-<dfn attribute for="Navigator" id="navigator-vrenabled-attribute">vrEnabled</dfn>
-The {{vrEnabled}} attribute's getter must return true if the context object is allowed to use the feature indicated by attribute name {{allowvr}} and VR is supported, and false otherwise.
+<dfn event for="VR" id="vr-displayconnect-event">displayconnect</dfn>
+A user agent MAY dispatch this event type to indicate that a {{VRDisplay}} has been connected.
+
+<dfn event for="VR" id="vr-displaydisconnect-event">displaydisconnect</dfn>
+A user agent MAY dispatch this event type to indicate that a {{VRDisplay}} has been disconnected.
+
+<dfn event for="VR" id="vr-navigate-event">navigate</dfn>
+A user agent MAY dispatch this event type to indicate that the current page has been navigated to from a page that was actively presenting VR content to the {{VRDisplay}}. The current page can call {{requestPresent()}} in response to this event in order to stay in VR presentation mode.
```

Comment:
It would be good to get @annevk's opinion on anything that ties into navigation. From a brief skim this seems redundant with the Window's load event, but I'm sure I'm missing something.